### DISCUSSION TITLE
Android Camera Permission

### DISCUSSION THREAD

**Discussion by blossomy** [yasirkula/UnityNativeCamera#88] [General]
I'm using this plugin, but I can't see the camera permission on my Android build.
It only asks about media files with a folder icon.
![Screen Shot 2022-03-07 at 3 27 37 PM](https://user-images.githubusercontent.com/12566316/157135937-f9553e1c-fdbc-409a-a194-b9b8bcc419c8.png)

I just unzipped your aar file so I saw the AndroidManifiest.xml.
There is no permission for "android.permission.CAMERA"

I think the permission should show this kind of camera icon.

<img width="457" alt="Screen Shot 2022-03-07 at 3 30 17 PM" src="https://user-images.githubusercontent.com/12566316/157135865-5daa0be3-35a7-431b-bdbd-ee8ddd3c2f0b.png">

**Comment by yasirkula** [comment]
Yes, camera permission isn't needed because the camera is accessed via the native camera app.

**Reply by blossomy** [reply]
We don't need to request the camera permission if the native camera app is used?
I thought the camera permission should be requested with any kind of camera.

**Reply by yasirkula** [reply]
Yes we don't need to request the permission. It's necessary only when you access and control the camera manually from your scripts (via native Android scripting) which isn't the case in NativeCamera.
